jdaviz/configs/cubeviz/helper.py

```diff
@@ -79,6 +80,28 @@ def load_data(self, data, data_label=None, override_cube_limit=False, **kwargs):
 
         super().load_data(data, parser_reference="cubeviz-data-parser", **kwargs)
 
+        if 'Spectral Extraction' not in self.plugins.keys():  # pragma: nocov
```

Comment:
I don't think you need `.keys()` here. Also I thought the directive is `no cover`, has that changed?

```suggestion
        if 'Spectral Extraction' not in self.plugins:  # pragma: no cover
```